pyproject.toml

```diff
@@ -108,7 +108,11 @@ min-public-methods=2
 
 [tool.isort]
 profile = "black"
-skip_glob = ["typings"]
+length_sort = true
```

Comment:
Is there any conflict between the isort and formater options?

Reply:
like black and isort?

Reply:
yep.

Reply:
```
[PASS] black format check passed!
11
Running isort format check...
12
ERROR: /home/runner/work/BentoML/BentoML/tests/integration/frameworks/test_pytorch_lightning_impl.py Imports are incorrectly sorted and/or formatted.
13
ERROR: /home/runner/work/BentoML/BentoML/tests/integration/frameworks/test_onnx_impl.py Imports are incorrectly sorted and/or formatted.
14
ERROR: /home/runner/work/BentoML/BentoML/tests/integration/frameworks/test_catboost_impl.py Imports are incorrectly sorted and/or formatted.
15
ERROR: /home/runner/work/BentoML/BentoML/tests/integration/frameworks/test_keras_impl.py Imports are incorrectly sorted and/or formatted.
16
ERROR: /home/runner/work/BentoML/BentoML/tests/integration/frameworks/test_lightgbm_impl.py Imports are incorrectly sorted and/or formatted.
```

Reply:
just tested. They're compatible with each other

Reply:
ok